sentry_sdk/integrations/gcp.py

```diff
@@ -143,6 +153,18 @@ def event_processor(event, hint):
 
         request["url"] = "gcp:///{}".format(environ.get("FUNCTION_NAME"))
 
+        if hasattr(gcp_event, "method"):
+            request["method"] = gcp_event.method
+
+        if hasattr(gcp_event, "query_string"):
+            request["query_string"] = gcp_event.query_string.decode("utf-8")
+
+        if hasattr(gcp_event, "headers"):
+            request["headers"] = _filter_headers(gcp_event.headers)
+
+        if hasattr(gcp_event, "data"):
```

Comment:
Please apply the same logic here as I commented on aws lambda. Do not attempt to json.loads unless you are sure it is json 100% of the time, attach raw data instead. Note though this is a Flask request object so the json property may be available too.

Reply:
Sure.